sk0-front/src/app/home/home.page.ts

```diff
@@ -1,32 +1,24 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
 
 import { IonicModule, NavController } from '@ionic/angular';
 import { AuthService } from '../auth/auth.service';
-import { HttpClientModule } from '@angular/common/http';
+import { CommonModule } from '@angular/common';
 
 @Component({
   selector: 'app-home',
   templateUrl: './home.page.html',
   styleUrls: ['./home.page.scss'],
   standalone: true,
-  imports: [IonicModule],
-  providers: [HttpClientModule]
+  imports: [IonicModule, CommonModule],
 })
-export class HomePage implements OnInit {
-  userName?: string;
+export class HomePage {
+  userName$ = this.authService.userName;
```

Comment:
Just for clarification. The reason we can directly refer to the userName in auth.service instead of "this.authService.userName.value" is because in home.page.html we use an async pipe for this variable: 
"h1 class="ion-no-margin">{{userName$ | async}}</h1"?

If we bound simply to userName$ without the async pipe in the HTML, we would need to refer to userName$ here as "this.authService.userName.value"?

Reply:
async pipe handles the subscription, while the `.value` only gets the actual state at the time when it was called. so when you emit a new value with `next()` then value will update with ` | async` only. and you can't use the `.value` for an observable